scapy/contrib/isotp.py

```diff
@@ -1911,7 +1891,9 @@ def ISOTPScan(sock, scan_range=range(0x7ff + 1), extended_addressing=False,
 
 
 def generate_text_output(found_packets):
-    if len(found_packets) != 0:
+    if len(found_packets) == 0:
+        return "No packets found."
+    else:
```

Comment:
This change was meant to reduce nesting.
The `else` should be removed.
Nesting is difficult for the human brain.

Reply:
done